clang/lib/DPCT/ValidateArguments.h

```diff
@@ -24,7 +24,7 @@
 /// two levels are supported currently.
 /// none:       uses helper functions from DPCT header files for memory
 ///             management migration
-/// restricted: uses API from DPC++ Explicit and Restricted Unified
+/// restricted: uses API from SYCL Explicit and Restricted Unified
```

Comment:
This text needs to be updated. it was valid, when USM was a DPC++ extension. Now it is part of SYCL 2020 standard and it doesn't have "explicit" and "restricted" levels, as far as I know.

Similar update is also needed for --usm-level=<value> option text.

Reply:
OK, I will fix it in a separate PR.